### DISCUSSION TITLE
(how to) expand "upcoming meetings" display to include next day?

### DISCUSSION THREAD

**Discussion by foundartfactory** [code4recovery/12-step-meeting-list#726] [Q&A] [answered]
Howdy! I hope this is the right place for this question :)

Over at [rockridgefellowship.org](https://rockridgefellowship.org) I'm using TSML, and the (initially hidden to me) Upcoming Meetings widget in the footer. If, say, the # of upcoming meetings to display is 3, but there are only 2 left today -- What's the best approach to have the filter roll over to the next day & display the third?

When I realized I was getting ready to edit filter_meetings.php I figured I should ask first ;)

Many thanks in advance for any insight --
Patrick

PS: huge thanks to all the Web Servants involved in the coe4recovery universe -- you have my gratitude, and that of a gazillion people over in my Oakland universe :-)

**Comment by brianw-area24** [comment, marked as answer]
I'm not sure I have a great answer for you. I think someone a while back looked into making that short code work the way you describe, but I think we ran into more issues that were hard to work out.

However, you may be interested in the new listing of meetings we're working on, TSML UI. It displays the next upcoming meeting first on the page. Currently you could create a page, and use the short code [tsml_ui]. However, we do have a member that's working on making this an option on the settings page, so you will be able to choose whether your meetings page uses the current way of showing meetings, or the new way. There are a few sites currently using the new way, and you can see a demo here: https://demo.code4recovery.org/tsml-ui

**Reply by foundartfactory** [reply]
Gotcha, and thanks for the info -- 

I'm **totally** interested in the TSML UI; I've been all over the repositories & at code4recovery -- didn't see an easy way to implement it at rockridgefellowship.org w/o hacking things up. :) I think the TSML UI looks great, so consider me excited!

Patrick
